File: Java/Baju.java
// Child dari Aksesoris
public class Baju extends Aksesoris {
    private String untuk;
    private String size;
    private String merk;

    public Baju(int id, String namaProduk, int hargaProduk, int stokProduk, 
                String jenis, String bahan, String warna, 
                String untuk, String size, String merk) {
        super(id, namaProduk, hargaProduk, stokProduk, jenis, bahan, warna);
        this.untuk = untuk;
        this.size = size;
        this.merk = merk;
    }

    @Override
    public void display() {
        super.display();
        System.out.printf(" %-8s | %-4s | %-10s |\n", untuk, size, merk);
    }
}

File: python/Baju.py
from AKsesoris import Aksesoris

class Baju(Aksesoris):
    def __init__(self, id, nama_produk, harga_produk, stok_produk, jenis, bahan, warna, untuk, size, merk):
        super().__init__(id, nama_produk, harga_produk, stok_produk, jenis, bahan, warna)
        self.untuk = untuk
        self.size = size
        self.merk = merk

    def get_info(self):
        return super().get_info() + [self.untuk, self.size, self.merk]